ops/charm.py

```diff
@@ -73,6 +73,12 @@ class LeaderSettingsChangedEvent(HookEvent):
     pass
 
 
+class CollectMetricsEvent(HookEvent):
+
+    def add_metrics(self, metrics, labels=None):
+        self.framework.model._backend.add_metric(metrics, labels)
```

Comment:
add_metrics vs. add_metric?  Which one is right?

Reply:
`add-metric` is the name of the hook tool.

I used `add_metrics` deliberately as it feels like that the public API should have the multiplicity of arguments in its name.

I can change it to be `add_metric` since you raised it.